LICENSE-3rdparty.csv

```diff
@@ -42,6 +42,7 @@ import,androidx.vectordrawable,Apache-2.0,Copyright 2018 The Android Open Source
 import,androidx.versionedparcelable,Apache-2.0,Copyright 2018 The Android Open Source Project
 import,androidx.viewpager,Apache-2.0,Copyright 2018 The Android Open Source Project
 import,androidx.work,Apache-2.0,Copyright 2018 The Android Open Source Project
+import,com.android.tools,GNU General Public License, Version 3, 29 June 2007. Copyright (C) 2007 Free Software Foundation
```

Comment:
The copyright notice here feels weird. I would expect the copyright to mention Google or Android, not the FSF

Reply:
there's no placeholder in their copyright so I don't really know exactly the format. Do you have another example somewhere ?

Reply:
nevermind I think I've found one